plugins/gateway/kong/plugin.go

```diff
@@ -190,11 +236,14 @@ func createTelemetry(kong *pdk.PDK) (*models.Telemetry, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get response headers: %v", err)
 	}
-	host, port, namespace := parseKongHost(routedService.Host)
+	host, port, _ := parseKongHost(routedService.Host)
 
+	// Note by Axel. Currently, we populate Host with (host+namespace), and don't put namespace on DestinationNamespace because
+	// on an issue on the way we create a service in ApiClarity either with newDiscoveredApi either with Telemetry
+	// Must be fixed with PA-11655
```

Comment:
Remove that comment

Reply:
hmmm... then in a general way, we need to define what kind of comments we want to put on the code: Here it is a comment that can help a future developer who will have to update this part of the code and can ask itself why namespace is not used here.
But I can remove if needed. (for me it's a kind of obfuscation)